src/QsCompiler/DataStructures/ReservedKeywords.fs

```diff
@@ -231,6 +231,11 @@ module GeneratedAttributes =
     let LoadedViaTestNameInsteadOf = "__LoadedViaTestNameInsteadOf__"
 
 
+/// contains reserved names for command line arguments
```

Comment:
Might be good to clarify that they are command-line arguments for the standalone executable specifically.

Reply:
Opposed to an executable with a driver? Standalone should be the default, so I don't think that should be added (the rest is interop capabilities, and not usually refered to as any kind of executable).

Reply:
Opposed to command-line arguments to the Q# command-line compiler, for example. It's not clear to me from the context in this file what "command-line arguments" refers to.

Reply:
edited to "contains reserved names for command line arguments of Q# projects"

Reply:
This change didn't go through. The latest version is still:

```
/// contains reserved names for command line arguments
```

Reply:
Oh, I found the change in #402 instead.